src/android/java/com/pspdfkit/cordova/action/document/ShowDocumentAction.java

```diff
@@ -95,39 +95,87 @@ private PdfActivityConfiguration parseOptionsToConfiguration(@NonNull final JSON
           builder.autosaveEnabled((Boolean) value);
         } else if ("backgroundColor".equals(option)) {
           builder.backgroundColor(Color.parseColor((String) value));
-        } else if ("disableAnnotationList".equals(option) && ((Boolean) value)) {
-          builder.disableAnnotationList();
+        } else if ("disableAnnotationList".equals(option)) {
+          if (((Boolean) value)) {
```

Comment:
If I'm seeing correctly, there's an extra parenthesis here and in few places below.

Reply:
Good catch 👍